src/components/PostEditor/PostEditor.js

```diff
@@ -696,7 +702,7 @@ export default class PostEditor extends React.Component {
               <LocationInput
                 saveLocationToDB
                 locationObject={curLocation}
-                location={location}
+                location={location || `${lat}, ${lng}`}
```

Comment:
If you setup the value for location in the prop you're sending in then I think this extra `||` should be able to be eliminated.